packages/react/src/components/stepper/Stepper.stories.tsx

```diff
@@ -0,0 +1,663 @@
+              if (index === state.activeStep - 1) {
+                return 'attention';
+              }
+              return stateName;
+            }),
+            fields: {
+              ...state.fields,
+              [action.fieldName]: {
+                value: action.newValue,
+                visited: true,
+              },
+            },
+          };
+        }
+
+        return {
+          activeStep: state.activeStep,
+          states: state.states.map((stateName, index) => {
+            if (index === state.activeStep - 1) {
+              return 'completed';
+            }
+            return stateName;
+          }),
+          fields: {
+            ...state.fields,
+            [action.fieldName]: {
+              value: action.newValue,
+              visited: true,
+            },
+          },
+        };
+      }
+      case 'completeStep': {
+        const activeStep = action.payload === 4 ? 4 : action.payload + 1;
+        return {
+          activeStep,
+          states: state.states.map((stateName, index) => {
+            if (index === action.payload - 1 && index !== 4 - 1) {
+              // current one but not last one
+              return 'completed';
+            }
+            if (index === action.payload) {
+              // next one
+              return 'available';
+            }
+            return stateName;
+          }),
+          fields: {
+            ...state.fields,
+          },
+        };
+      }
+      case 'setActive': {
+        if (!activeStepIsValid(state) && !weAreInLastAvailableStep(state)) {
+          return {
+            activeStep: state.activeStep,
+            states: state.states.map((stateName, index) => {
+              if (index === state.activeStep - 1) {
+                return 'attention';
+              }
+              return stateName;
+            }),
+            fields: {
+              ...state.fields,
+            },
+          };
+        }
+
+        return {
+          activeStep: action.payload,
+          states: state.states.map((stateName, index) => {
+            if (index === action.payload - 1 && stateName !== 'attention' && stateName !== 'paused') {
+              return 'available';
+            }
+            if (index === state.activeStep - 1 && activeStepIsValid(state)) {
+              return 'completed';
+            }
+            return stateName;
+          }),
+          fields: {
+            ...state.fields,
+          },
+        };
+      }
+      default:
+        throw new Error();
+    }
+  };
+
+  const initialState = {
+    activeStep: 1,
+    states: ['available', 'disabled', 'disabled', 'disabled'],
+    fields: {
+      firstName: {
+        value: '',
+        visited: false,
+      },
+      lastName: {
+        value: '',
+        visited: false,
+      },
+      age: {
+        value: undefined,
+        visited: false,
+      },
+    },
+  };
+  const labels = ['First name', 'Last name', 'Age', 'Verify and send'];
```

Comment:
"Verify and send" is often called "Review and send". I think "review" is slightly more understandable and less engineer way of saying it.

Reply:
- [ ] Refactor: Verify and send -> Review and send